file-formats/smbc/type/zif_aff_smbc_v1.intf.abap

```diff
@@ -118,68 +118,84 @@ INTERFACE zif_aff_smbc_v1
+      entity_set                     TYPE string,
       "! <p class="shorttext synchronized">Section Layout</p>
       "! Toggle between onepage concept and tabs.
       "! $values {@link zif_aff_smbc_v1.data:co_section_layout}
       "! $default {@link zif_aff_smbc_v1.data:co_section_layout.page}
-      section_layout          TYPE string,
+      section_layout                 TYPE string,
       "! <p class="shorttext synchronized">Variant Management</p>
       "! $values {@link zif_aff_smbc_v1.data:co_page_variant_management}
       "! $default {@link zif_aff_smbc_v1.data:co_page_variant_management.none}
-      variant_management      TYPE string,
+      variant_management             TYPE string,
       "! <p class="shorttext synchronized">Editable Header Content</p>
       "! If true, the header content is changeable in edit mode
-      editable_header_content TYPE abap_bool,
+      editable_header_content        TYPE abap_bool,
+      "! <p class="shorttext synchronized">Show Transport Selection Strip</p>
+      "! If true, the selected transport request and transport request select action are displayed in a message strip.
+      show_transport_selection_strip TYPE abap_bool,
     END OF ty_object_page,
     "! <p class="shorttext synchronized">Table Settings</p>
     BEGIN OF ty_table_setting,
       "! <p class="shorttext synchronized">Entity Set</p>
-      entity_set             TYPE string,
-      table_type             TYPE ty_table_type,
+      entity_set                    TYPE string,
+      table_type                    TYPE ty_table_type,
       "! <p class="shorttext synchronized">Selection Mode</p>
       "! $values {@link zif_aff_smbc_v1.data:co_selection_mode}
-      "! $default {@link zif_aff_smbc_v1.data:co_selection_mode.auto}
-      selection_mode         TYPE string,
+      "! $default {@link zif_aff_smbc_v1.data:co_selection_mode.multi}
+      selection_mode                TYPE string,
+      "! <p class="shorttext synchronized">Select All</p>
+      "! Enable Select all checkbox
+      select_all                    TYPE abap_bool,
       "! <p class="shorttext synchronized">Row Count Mode</p>
       "! Defines how the table handles the visible rows in the table
       "! $values {@link zif_aff_smbc_v1.data:co_row_count_mode}
       "! $default {@link zif_aff_smbc_v1.data:co_row_count_mode.fixed}
-      row_count_mode         TYPE string,
+      row_count_mode                TYPE string,
       "! <p class="shorttext synchronized">Row Count</p>
       "! Number of visible rows of the table
       "! $minimum 1
       "! $maximum 1000
       "! $default '5'
-      row_count              TYPE i,
+      row_count                     TYPE i,
+      "! <p class="shorttext synchronized">Frozen Column Count</p>
+      "! You can freeze the first columns of a table so that they always remain visible when scrolling the table horizontally
+      "! $minimum 0
+      "! $maximum 30
+      "! $default '0'
```

Comment:
Minor: I guess no default would be needed since 0 is automatically the default value for integers.

Reply:
ABAP lint and default value -> done